public/app/features/alerting/unified/RuleEditor.tsx

```diff
@@ -1,6 +1,63 @@
-import React, { FC } from 'react';
+import { Alert, Button, InfoBox, LoadingPlaceholder } from '@grafana/ui';
+import Page from 'app/core/components/Page/Page';
+import { useCleanup } from 'app/core/hooks/useCleanup';
+import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
+import { RuleIdentifier } from 'app/types/unified-alerting';
+import React, { FC, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import { AlertRuleForm } from './components/rule-editor/AlertRuleForm';
+import { useUnifiedAlertingSelector } from './hooks/useUnifiedAlertingSelector';
+import { fetchExistingRuleAction } from './state/actions';
+import { parseRuleIdentifier } from './utils/rules';
 
-const RuleEditor: FC = () => <AlertRuleForm />;
+const ExistingRuleEditor: FC<{ identifier: RuleIdentifier }> = ({ identifier }) => {
```

Comment:
nit: I think introducing a Prop type or interface would be nice here.

Reply:
- [ ] add Prop type